source/includes/reference/_borderless.md

```diff
@@ -442,13 +524,51 @@ X-idempotence-uuid                | Unique identifier assinged by you. Used for
 
 
 
-## Get Available Currencies
+## Get Available Currency Codes
+
+Get list of currencies that your can hold in your borderless account.
 
 > Example Request:
 
 ```shell
-curl -X GET https://api.sandbox.transferwise.tech/v1/borderless-accounts/balance-currencies \
+curl -X GET https://api.sandbox.transferwise.tech/v2/borderless-accounts-configuration/profiles/<your_profile_id>/available-currencies \
      -H "Authorization: Bearer <your api token>" 
+     
+```
+
+> Example Response:
+
+```json
+[
+    "EUR",
+    "GBP",
+    "USD",
+...
+    "UAH",
+    "ARS",
+    "NPR"
+]
+```
+
+#### Request
+
+**`GET https://api.sandbox.transferwise.tech/v2/borderless-accounts-configuration/profiles/<your_profile_id>/available-currencies`**
+
+#### Response
+
+Array of 3-letter currency codes according to the [ISO 4217](https://www.iso.org/iso-4217-currency-codes.html) 
+
+## Get Available Currency Information
+
+Get list of currencies that your can hold in your borderless account. Also shows which currencies have the option to get bank account details.
```

Comment:
List the currencies that **you** can enable in your borderless account. It also shows which currencies support bank account details.